.github/workflows/ci.yml

```diff
@@ -0,0 +1,15 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    - name: Run Checks
```

Comment:
and then you can add a cache here, as the project is very small, it will work:

```
- name: Cache dependencies
  uses: actions/cache@v1
  with:
    path: .gradle/caches
    key: gradle-cache-${{ hashFiles('**/*.gradle') }}
    restore-keys:   gradle-cache
```